commands/pull.js

```diff
@@ -138,32 +149,39 @@ const run = co.wrap(function * (sourceIn, targetIn) {
 function * push (context, heroku) {
   const fetcher = require('../lib/fetcher')(heroku)
   const {app, args} = context
+  const flags = context.flags
+  const exclusions = parseExclusions(flags['exclude'])
```

Comment:
@BenTalagan just noticed one more thing: this should have been changed to `exclude-table-data` as well in https://github.com/heroku/heroku-pg/pull/160/commits/8a780ace99682c4d67312579058cedc6537e429a right?

Reply:
Damn you're right, thank you for noticing!